client/src/ui/components/friends/ChatWindow.tsx

```diff
@@ -1,15 +1,33 @@
 import { Chat } from "../../../types/Chat";
 import React, { useEffect, useState } from "react";
 import windowResize from "../../utils/WindowResize";
+import Title from "../common/Title";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import { receivedMessageBox, sentMessageBox } from "./MessageBox";
+import ChatInputBox from "./ChatInputBox";
```

Comment:
prob should've enforced this before but it'd be nice to keep imports grouped up to make it clean

one thing easy to do is keep the absolute imports (from libraries/modules) at the top and then relative imports (other components/functions in this app) below, so for example
```
import React, { useEffect, useState } from "react";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faEdit } from "@fortawesome/free-solid-svg-icons";

import { receivedMessageBox, sentMessageBox } from "./MessageBox";
import ChatInputBox from "./ChatInputBox";
import windowResize from "../../utils/WindowResize";
import Title from "../common/Title";
```
(also note the two `fortawesome` imports are beside each other)